### PR TITLE
Fix/automating popover menu docs

### PR DESCRIPTION
**Pull request by nitayneeman**
Resolves wix-private/wsr-issues#302

### CODE REVIEW

File: src/PopoverMenu/PopoverMenu.js [hunk 1]
```diff
@@ -25,19 +46,11 @@ class PopoverMenu extends WixComponent {
     buttonTheme: 'icon-greybackground',
     buttonHeight: 'medium',
     maxWidth: '378px',
+    appendToParent: false,
+    zIndex: 0,
```
Comment (hepiyellow) [inline src/PopoverMenu/PopoverMenu.js]: @nitayneeman Why did you add default `zIndex: 0`?
This `zIndex` goes to `<Tooltip/>`, which puts it on the `<TooltipContent/>`.
The tooltip has a default zIndex which should be the highest of all components (currently 2000).

Setting it to 0, means it could be (will be) under most other part/components of the page.

Comment (nitayneeman) [reply src/PopoverMenu/PopoverMenu.js]: I agree with your point and I guess that part of the code shouldn't be merged (probably was just for checking a specific case momently). Moreover, the whole PR focused on improving the relevant docs, without changing the component implementation. Anyway, this code is redundant.
